src/util/antlr/ANTLRErrorHandling.h

```diff
@@ -4,21 +4,54 @@
 
 #pragma once
 
+#include <parser/ParseException.h>
+
 #include <string>
 
-#include "../../parser/ParseException.h"
 #include "BaseErrorListener.h"
+#include "CharStream.h"
+#include "CommonTokenStream.h"
 #include "DefaultErrorStrategy.h"
+#include "ParserRuleContext.h"
 #include "RecognitionException.h"
+#include "Recognizer.h"
+#include "TokenSource.h"
+
+// TODO<qup42> whats up with this namespace?
+namespace {
+[[maybe_unused]] std::string colorError(const ExceptionMetadata& metadata) {
```

Comment:
If this is a member function of `ExceptionMetadata` and is called `coloredErrorMessage`,

then `LOG(ERROR) << metadata.coloredErrorMessage() << std::endl;` is self-documenting code.